kubemarine/procedures/upgrade.py

```diff
@@ -27,6 +26,13 @@
 from kubemarine.procedures import install
 
 
+def cleanup_tmp_dir(cluster: KubernetesCluster) -> None:
+    # Clean up kubernetes tmp dir, where backup files from previous upgrades are located
+    nodes = cluster.make_group_from_roles(roles=["control-plane", "worker"])
+    nodes.sudo("rm -rf /etc/kubernetes/tmp")
```

Comment:
It's a removing, not cleanup, since target directory is being deleted

Reply:
command was fixed to not remove directory